quizes/quizInstructions.css

```diff
@@ -0,0 +1,17 @@
+.rules-body{
+    align-items: center;
+    border: 2px solid var(--white);
+    border-radius: var(--gutter-one);
+    display: flex;
+    flex-direction: column;
+    height: var(--card-large);
+    margin: auto;
+    width: var(--card-large);
+}
+.rules{
+    color: var(--white);
+    line-height: var(--gutter-two);
```

Comment:
Instead of naming as gutter , name it in terms of spacing as gutter is not generic and maybe cause confusion later.

Reply:
yeah, You are right. Done✅.